fmn/consumer/worker.py

```diff
@@ -54,6 +71,16 @@ def get_preferences():
 
 
 def update_preferences(openid, prefs):
+    """
+    Update an existing preference dictionary loaded by :func:`get_preferences`.
```

Comment:
I'd have the same comment as above here, I'd specify we're updating just a part of the dict :)